.github/workflows/test-and-build-workflow.yml

```diff
@@ -0,0 +1,30 @@
+name: Notifications Test and Build
```

Comment:
license header?
Also "---" at the beginning of file

Reply:
done